### PR TITLE
Graph network dropdown

### PR DESCRIPTION
**Pull request by lubej**

### CODE REVIEW

**Comment by lukaw3d** [comment]
This has a bit of an issue with keyboard accessibility
| Before | After |
| --- | --- |
| tab-before  | tab  |

File: src/app/pages/HomePage/Graph/NetworkSelector/index.tsx [hunk 1]
```diff
@@ -1,100 +1,133 @@
...
+
+const StyledOption = styled(StyledSelectOption)(() => ({
+  height: '44px',
+  color: COLORS.grayDark,
+  svg: {
+    color: COLORS.grayExtraDark,
+  },
+  [`&:hover:not(.${optionClasses.disabled}),
+  &.${optionClasses.highlighted}`]: {
+    backgroundColor: 'transparent',
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/NetworkSelector/index.tsx:60]: Did you mean to give it some color when hovered / focused?
```suggestion
  [`&:hover:not(.${optionClasses.disabled}),
  &:focus-visible:not(.${optionClasses.disabled}),
  &.${optionClasses.highlighted}`]: {
    backgroundColor: 'red',
```

File: src/app/pages/HomePage/Graph/NetworkSelector/index.tsx [hunk 2]
```diff
@@ -1,100 +1,133 @@
...
   backgroundColor: theme.palette.layout.primaryBackground,
-  border: `solid 3px ${theme.palette.layout.lightBorder}`,
-  borderRadius: '45px',
-  padding: theme.spacing(3, 4),
-  display: 'inline-flex',
-  alignItems: 'center',
+  border: `2px solid ${theme.palette.layout.lightBorder}`,
+  color: theme.palette.layout.main,
+  '&:hover': {
+    backgroundColor: theme.palette.layout.primaryBackground,
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/NetworkSelector/index.tsx:42]: Did you mean to change to different bg on hover/focus?
```suggestion
  '&:hover, &:focus-visible': {
    backgroundColor: 'red',
```

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/HomePage/Graph/NetworkSelector/index.tsx (hunk 1 above)
Comment (lubej) [reply src/app/pages/HomePage/Graph/NetworkSelector/index.tsx:60]: Right, thanks for noticing.

**Pull request merged**